app/src/main/java/org/wikipedia/page/PageActivity.kt

```diff
@@ -664,23 +666,20 @@ class PageActivity : BaseActivity(), PageFragment.Callback, LinkPreviewDialog.Ca
         if (!Prefs.showOneTimeCustomizeToolbarTooltip) {
             return
         }
-        val anchorView: View?
-        var aboveOrBelow = true
-        if (Prefs.customizeToolbarMenuOrder.contains(PageActionItem.THEME.id)) {
-            anchorView = binding.pageToolbarButtonShowOverflowMenu
-            aboveOrBelow = false
-        } else {
-            anchorView = pageFragment.getPageActionTabLayout().children.find { it.id == PageActionItem.THEME.hashCode() }
-        }
-        anchorView?.let {
-            it.postDelayed({
-                if (!isDestroyed) {
-                    FeedbackUtil.showTooltip(this, it, getString(R.string.theme_chooser_menu_item_tooltip),
-                            aboveOrBelow = aboveOrBelow, autoDismiss = false, -DimenUtil.roundedDpToPx(8f), 0)
-                }
-            }, 2000)
-        }
-        Prefs.showOneTimeCustomizeToolbarTooltip = false
+        binding.pageToolbarButtonShowOverflowMenu.postDelayed({
+            if (!isDestroyed) {
+                themeTooltipBalloon =
+                    FeedbackUtil.getTooltip(this, getString(R.string.theme_chooser_menu_item_short_tooltip), arrowAnchorPadding = -DimenUtil.roundedDpToPx(12f), topOrBottomMargin = -12, aboveOrBelow = true, autoDismiss = false, showDismissButton = true, callback = object :
+                        FeedbackUtil.Callback {
+                        override fun onDismissClicked() {
+                            Prefs.showOneTimeCustomizeToolbarTooltip = false
+                            Prefs.toolbarTooltipVisible = false
+                        }
```

Comment:
Would it be possible to just use `themeTooltipBalloon.setOnBalloonDismissListener { }`  instead of creating a new interface?